### PR TITLE
Bad rendering when attempting to remove a blockquote - Iss187

### PR DESCRIPTION
**Pull request by lucasferreiralimax**
## Problem

[Bad rendering when attempting to remove a blockquote #187](https://github.com/globocom/megadraft/issues/187)
When you clear the editor and start typing something show a text placeholder
But if you use another type of text has a bug
___
## Reproduce

Problem
___
## Solution

[Draft.js](https://draftjs.org) default hides text placeholder with a class and doesn't has spacing style
But, I created a switch case to push a class on each type with a position

Solution

### CODE REVIEW

**Comment by lucasferreiralimax** [comment]
@JeanPinzon

File: src/components/MegadraftEditor.js [hunk 1]
```diff
@@ -387,10 +387,39 @@ export default class MegadraftEditor extends Component {
...
+          classEditor += " placeholder-header-two";
+          break;
+        case "blockquote":
+          classEditor += " placeholder-blockquote";
+          break;
+        default:
+          classEditor += "";
+          break;
+      }
+    }
```
Comment (JeanPinzon) [inline src/components/MegadraftEditor.js]: @lucasferreiralimax, what do you think about create a method to abstract this logic?

File: src/components/MegadraftEditor.js [hunk 2]
```diff
@@ -387,10 +387,39 @@ export default class MegadraftEditor extends Component {
...
+          classEditor += " placeholder-list";
+          break;
+        case "header-two":
+          classEditor += " placeholder-header-two";
+          break;
+        case "blockquote":
+          classEditor += " placeholder-blockquote";
+          break;
+        default:
+          classEditor += "";
```
Comment (JeanPinzon) [inline src/components/MegadraftEditor.js]: I think it is not necessary.

Comment (lucasferreiralimax) [reply src/components/MegadraftEditor.js]: Yes, i cleaned code and remove this default in switch

File: src/components/MegadraftEditor.js (hunk 1 above)
Comment (lucasferreiralimax) [reply src/components/MegadraftEditor.js]: I created the function for manipulate and to abstract logic also adjustment an class with an identifier with template string.

See the work @JeanPinzon

**Pull request merged**